app/src/main/java/com/oracle/javafx/scenebuilder/app/menubar/MenuBarController.java

```diff
@@ -1171,6 +1175,23 @@ public String getTitle() {
         insertMenu.setOnMenuValidation(onCustomPartOfInsertMenuValidationHandler);
         
         windowMenu.setOnMenuValidation(onWindowMenuValidationHandler);
+
+        // Modify the Check Updates menu item if there is an update available.
+        // Icons by Font Awesome (https://fontawesome.com/license/free) under CC BY 4.0 License
+        AppSettings.getLatestVersion(latestVersion -> {
+            if (AppSettings.isUpdateAvailable()) {
+                Image icon = new Image(MenuBarController.class
+                        .getResource("download_icon.png")
+                        .toExternalForm());
+                ImageView iconView = new ImageView(icon);
+
+                Platform.runLater(() -> {
+                    checkUpdatesMenuItem.setGraphic(iconView);
+                    checkUpdatesMenuItem.disableProperty().setValue(false);
```

Comment:
Does it make sense to disable this menu option? 

In case the user really wanted to be sure, clicking on it should at least bring a dialog saying that the current version XXX is already up-to-date, which is what we already do now?

<img width="381" alt="image" src="https://user-images.githubusercontent.com/2043230/155957479-9b05f2b9-807d-4a21-ab05-c05981577a55.png">